source/administration/plugins.rst

```diff
@@ -19,6 +19,8 @@ There are two main types of plugins, both of which are supported in Team and Ent
 
 Later in upcoming releases, the `Enterprise Edition <https://about.mattermost.com/pricing>`_ will also support plugins that enable you to extend Mattermost functionality to meet a specific, complex requirement such as profiling performance metrics, and to implement highly customized compliance rules for `information barriers <http://www.17a-4.com/supervision-information-barriers/>`_.
 
+For a list of available plugins, including those for auto-linking, auto-censoring and anti-virus scanning, as well as third-party plugins with JIRA, Zoom, GitHub and more, see the `Mattermost plugins repo <https://github.com/mattermost/mattermost-plugins>`_. 
```

Comment:
Do we want to link them to the plugins repo and the apps available from the Webpage (https://about.mattermost.com/community-applications/)

Reply:
For now just the plugins repo - the apps available on the Webpage are largely for webhooks/slash commands. We link to the Webpage on those integration docs, e.g. https://docs.mattermost.com/developer/webhooks-incoming.html#share-your-integration

Eventually all the plugins and webhook integrations should be in one place hopefully with the new app directory you're working on :)